asv_bench/benchmarks/groupby.py

```diff
@@ -1535,7 +674,9 @@ def setup(self):
         self.secid_max = int('F0000000', 16)
         self.step = ((self.secid_max - self.secid_min) // (self.n_securities - 1))
         self.security_ids = map((lambda x: hex(x)[2:10].upper()), range(self.secid_min, (self.secid_max + 1), self.step))
-        self.data_index = MultiIndex(levels=[self.dates.values, self.security_ids], labels=[[i for i in range(self.n_dates) for _ in xrange(self.n_securities)], (range(self.n_securities) * self.n_dates)], names=['date', 'security_id'])
+        self.data_index = MultiIndex(levels=[self.dates.values, self.security_ids],
+                                     labels=[[i for i in range(self.n_dates) for _ in xrange(self.n_securities)], (range(self.n_securities) * self.n_dates)],
```

Comment:
@jorisvandenbossche  Looks like I missed this in the first pass, but this ```xrange``` call will break things on Py3. Compatability is being handled by overwriting ```range``` in ```pandas_vb_common```.